src/EditorFeatures/Core.Wpf/InlineHints/InlineHintsTagger.cs

```diff
@@ -171,28 +173,40 @@ public IReadOnlyList<TagSpan<IntraTextAdornmentTag>> GetTags(NormalizedSnapshotS
                 var classify = document != null && _taggerProvider.EditorOptionsService.GlobalOptions.GetOption(InlineHintsViewOptionsStorage.ColorHints, document.Project.Language);
 
                 var selectedSpans = new List<TagSpan<IntraTextAdornmentTag>>();
-                for (var i = 0; i < _cache.Count; i++)
+                var itemsToUpdate = new List<(IMappingTagSpan<InlineHintDataTag> mappingTagSpan, TagSpan<IntraTextAdornmentTag> hintTagSpan)>();
```

Comment:
why do we need mapping tag spans?

Reply:
GetTags on the tagaggregator retuens mappingtagspans

Reply:
this goes back to our earlier conversation.  I don't know why we're even using tag aggregators here.